pkgs/top-level/python-packages.nix

```diff
@@ -27603,23 +27603,25 @@ EOF
   trezor_agent = buildPythonPackage rec{
     name = "${pname}-${version}";
     pname = "trezor_agent";
-    version = "0.9.0";
+    version = "v0.9.2";
```

Comment:
if you need the `v` you can add it to `rev`. It shouldn't be here because this is part of the name.